spec/rspec/rails/example/system_example_group_spec.rb

```diff
@@ -18,6 +18,33 @@ module RSpec::Rails
           end
         end
       end
+
+      describe '#driver' do
+        it 'uses :selenium driver by default' do
+          group = RSpec::Core::ExampleGroup.describe do
+            include SystemExampleGroup
+          end
+          group.new.driver
+
+          expect(Capybara.current_driver).to eq :selenium
+        end
+
+        it 'sets :rack_test driver using by before_action' do
+          group = RSpec::Core::ExampleGroup.describe do
+            include SystemExampleGroup
+
+            before do
+              driven_by(:rack_test)
+            end
+          end
+          example = group.new
+          allow(example).to receive(:driven_by).and_call_original
+          group.hooks.run(:before, :example, example)
+
+          expect(Capybara.current_driver).to eq :rack_test
+          expect(example).to have_received(:driven_by).once
```

Comment:
Those two balancing examples look way too different to be able to understand between example group definitions and resulting behaviour.